ui/packages/consul-ui/app/components/consul/bucket/list/index.hbs

```diff
@@ -19,11 +19,23 @@
       <dd>
         {{@item.Namespace}}
       </dd>
+
+{{#if (and @service @item.Service)}}
```

Comment:
Is this indentation intentional?

Reply:
I 'think' this is in reference to the two different indentation level thing I do quite often? If it is, then yeah its intentional. In Consul you'll often find files that have this idea of two levels of indentation. The HTML-y bits have their own indention which grows to the right the more nested it gets, as you would expect, I sometimes then put the logical handlebars-y stuff over to the left. In my eyes when scanning over a file quickly it's less dense and far easier to figure out whats going on. I know some folks find it strange 😅, but it works.

Reply:
Cool, just checking it wasn’t an auto-formatter like Prettier making mischief 🙂

Reply:
nah but as you can imagine it will make mischief once we come to use prettier on the templates 😂  I guess the advantage will be formatting just happens so it will be a case of out with the old in with the new 😄